spec/factories/messages.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :message do
+    
```

Comment:
Will add a commit with a factory for messages, I just wanted to get the model out as soon as possible as other issues rely on it.